internal/logger/logger_test.go

```diff
@@ -50,20 +50,21 @@ func init() { RegisterTestSuite(&LoggerTest{}) }
 // Boilerplate
 // //////////////////////////////////////////////////////////////////////
 
+func createLoggerThatWritesToGivenBuffer(buf *bytes.Buffer, level config.LogSeverity) {
```

Comment:
The name is a bit misleading. It's not a new logger. Well technically it is a new object, but in use it's just replacing/updating the default-logger. 
Let's rename this to something like `redirectLogsToGivenBuffer` .